aiida_workgraph/engine/workgraph.py

```diff
@@ -369,22 +394,34 @@ def _on_awaitable_finished(self, awaitable: Awaitable) -> None:
         """
         print("on awaitable finished: ", awaitable.key)
         self.logger.info(
-            "received callback that awaitable %d has terminated", awaitable.pk
+            "received callback that awaitable %d has terminated", awaitable.key
```

Comment:
pk is the AiiDA Process identifier, and key is the Awaitable identifier? Do you think it might be useful to put in both information? Something like
```suggestion
            "received callback that awaitable %d started in process pk<%d> has terminated", awaitable.key, awaitable.pk
```

Reply:
I put both there if the pk and key are not the same.